<?xml version="1.0"?>
<launch>
  <arg name="hardware_interface" default="PositionJointInterface"/>
  <arg name="controllers" default="joint_state_controller pos_joint_trajectory_controller"/>
  <arg name="robot_name" default="iiwa" />
  <arg name="model" default="iiwa7" />

  <group ns="$(arg robot_name)">
    <node name="iiwa_kinematics" pkg="iiwa_control" type="iiwa_kinematics.py" output="screen"/>
  </group>
</launch>
